### DISCUSSION TITLE
How to set "BELLMAN_CUSTOM_GPU"?

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#9409] [Build and Sync] [answered]
I have two video cards. How can I designate one to use? "BELLMAN_ CUSTOM_ Is GPU "set? If yes, how to obtain the value of this configuration?

**Comment by rjan90** [comment, marked as answer]
Hey!

You will have to set `CUDA_VISIBLE_DEVICES = "x"` before running a worker to designate a single GPU to a `lotus-worker`. Also make sure to use separate repos for each worker.

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
and tmp dirs
